RInAppMessaging/Classes/IAMPreferenceBuilder.swift

```diff
@@ -25,11 +25,19 @@
         return self
     }
 
+    @objc @discardableResult
+    public func setIDTrackingIdentifier(_ idTrackingIdentifier: String?) -> IAMPreferenceBuilder {
```

Comment:
Can the param be non-optional?

Reply:
I think we can. I just tried to be consistent with other setters

Reply:
Ok, updated. Other optionals will be removed as a part of UserInfo refactoring

Reply:
Ah it was for consistency. You can keep it as optional until they are all changed at same time. Please add it as a task to tech debt page.

Reply:
Added to tech debt log